### DISCUSSION TITLE
Checkbox as textbox in link expanded form [Aug 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3141] [Discord Threads]
> 🙋‍♂️ **alex.pro**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### it's a bug. Help us with details here   
#### https://github.com/nocodb/nocodb/issues  
#  
> 🙋‍♂️ **alex.pro**  
#### https://github.com/nocodb/nocodb/issues/3018  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1005007597453189171)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
